src/pages/Main/CanvasDiv.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { canvasElement } from "./Canvas";
+import './CanvasDiv.css';
+
+interface CanvasDivProps {
+    
+}
+
+const CanvasDiv: React.FC<CanvasDivProps> = () => {
+  const canvasContainerRef = React.useRef(document.createElement("div"));
+
+  React.useEffect(()=>{
+    canvasContainerRef.current.appendChild(canvasElement);
+  }, [])
+
+  return <div className='canvasDiv' ref={canvasContainerRef}></div>
+}
+
+export { CanvasDiv };
+
```

Comment:
Would you move this file and CanvasDiv.css (maybe renamed to CanvasWrapper or CanvasContainer) out of Main and into a new folder under components called Canvas?

Reply:
ok . I also named  the folder canvasContainer to avoid confusion with the canvas folder under src that contains canvas.tsx.